src/cascadia/TerminalApp/App.cpp

```diff
@@ -632,12 +271,16 @@ namespace winrt::TerminalApp::implementation
         //  - display a loading error
         _settingsLoadedResult = _TryLoadSettings(false);
 
+        auto terminalPage = _root.as<TerminalPage>();
+        // Update the settings in TerminalPage
+        terminalPage->SetSettings(_settings.get());

```

Comment:
Should this be after the if(FAILED) block below?  Don't set it if it didn't successfully load?

Reply:
Moved below if(FAILED), sorry for this mistake